api-reference/10 UI Components/dxOverlay/1 Configuration/copyRootClassesToWrapper.md

```diff
@@ -5,7 +5,29 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Copies user-defined classes from the root element to the wrapper element. 
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+This property allows you to configure a single {widgetName}'s wrapper element to customize shade, resize, or drag zone.
```

Comment:
```suggestion
This property allows you to use CSS for wrapper element configuration. You can customize shade, resize, or drag zone.
```

> customize shade, resize, or drag zone.

What does that mean?

Reply:
something like this:
![image](https://user-images.githubusercontent.com/33040934/127663129-a3170751-d659-497e-b1ee-4ed2ab0cfaef.png)

Reply:
Okay, that's for shade customization. What about the rest ("resize, or drag zone")?

Reply:
Seems like nobody except Anton knows these scenarios. He definitely said that 'the wrapper is responsible for shade, resize and drag zone', but I cannot request clarification since he took a personal day.